sgid/index.html

```diff
@@ -0,0 +1,111 @@
+        <p class="section-read-more">
+            <a href="https://utah.maps.arcgis.com/home/groups.html" class="btn">Browse ArcGIS Online</a>
+        </p>
+    </div>
+
+    <div class="grid__col section">
+        <h2>SGID Open Data Site</h2>
+        <p class="readable">The <a href="https://opendata.gis.utah.gov">SGID Open Data site</a> leverages Esri's Open Data platform to create a single portal for GIS resources hosted in ArcGIS Online or ArcGIS Enterprise Portal. Each item in Open Data contains web service links for consuming the datasets in webmaps and other applications along with data download links. We share all our curated datasets through the site and encourage other agencies and organizations to <a href="{% link gis-data-sharing/open-data/index.md %}">share</a> their datasets here as well to help us spread GIS data to the widest possible audience.</p>
+        <p class="section-read-more">
+            <a href="https://opendata.gis.utah.gov" class="btn">Browse Open Data</a>
+        </p>
+    </div>
+
+    <div class="grid__col section alternate">
+        <h2>The Raster Discovery App</h2>
+        <div class="grid__col--2-of-2 grid__col--centered grid__col--s-hidden" style="width:300px">
+            <a href="https://raster.utah.gov">
+                <img alt="Raster Discover App" class="text-center" src="{% link images/applications/raster_utah_gov.png %}">
+            </a>
+        </div>
+        <p class="readable">The Raster Discovery App allows users to search all of AGRC's raster datasets, including publicly-available imagery, elevation, lidar, and scanned maps. We have tiled many of these datasets to provide better access to individual areas, and the app allows you to search for and download only the tiles that cover your area of interest.</p>
+        <p class="section-read-more">
+            <a href="https://raster.utah.gov" class="btn">Search Raster Data</a>
+        </p>
+    </div>
+
+    <div class="grid__col section">
+        <h2>Discover Basemap and Imagery Services</h2>
+        <div class="grid__col--span-all grid__col--centered grid__col--s-hidden" style="width:485px">
+            <a href="{% link discover/index.html %}">
+                <img alt="Discover Basemap and Imagery Services" class="text-center" src="{% link /discover/images/discover.png %}">
+            </a>
+        </div>
+        <p class="readable">Discover, an implementation of AppGeo's <a href="https://www.appgeo.com/giza/">Giza</a> platform, serves up web services of our more popular raster products. These include both public-access and licensed aerial imagery products as well as the high-quality, Utah-specific basemaps created by AGRC's cartographers. Discover serves all these layers as WMS image and WMTS tiled image services that anyone can use in a wide variety of desktop and web applications.</p>
```

Comment:
**suggestion**: remove what and use the language from the discover page to tell why.
```suggestion
        <p class="readable">Discover is a cloud-based server that provides AGRC's aerial imagery and base maps to anyone who wants to use them in their GIS, CAD, or web applications. These include both public-access and licensed aerial imagery products as well as the high-quality, Utah-specific basemaps created by AGRC's cartographers. Discover serves all these layers as WMS image and WMTS tiled image services that anyone can use in a wide variety of desktop and web applications.</p>
```

Reply:
I want to give AppGeo a shoutout here. We mention Esri several times, so it seems only fair that we give AppGeo credit too.

Reply:
I'd prefer we remove as many references to esri as possible. If you want to keep it i'd suggest moving it to the discover page as someone familiarizing themself with the sgid is most likely not interested in this information.

Reply:
I guess I use "Esri" to imply "this many only apply to you if you're working in the Esri ecosystem." I'll see if there's a better way to convey this.